### ISSUE TITLE
Add texture to GLTF

### ISSUE THREAD

**Issue by nampereira** [conix-center/ARENA-core#340]
While it might not make sense for every GLTF (they can be a complex hierarchy of objects), it seems that it could be useful to be able to apply a texture to a GLTF.  I think we can add this to material-extras. 

Some pointers here:
https://github.com/n5ro/aframe-extras/issues/167

**Comment by nampereira** [comment]
Addressed by #343.

Set `overrideSrc` in `material-extras` to the desired texture URL:
![image](https://user-images.githubusercontent.com/3504501/134255837-1a55b214-dbf7-44b6-976b-3f2e2dd37632.png)

Here is an example json where we apply  `overrideSrc` to a gltf:
```
{
  "object_id": "Pumpkin",
  "persist": true,
  "type": "object",
  "action": "update",
  "data": {
    "object_type": "gltf-model",
    "url": "/store/users/nuno/Pumpkin/Pumpkin.gltf",
    "position": {
      "x": 1,
      "y": 2,
      "z": -2
    },
    "rotation": {
      "x": 0,
      "y": 180,
      "z": 0
    },
    "scale": {
      "x": 0.6,
      "y": 0.6,
      "z": 0.6
    },
    "material-extras": {
      "overrideSrc": "/store/users/nuno/Pumpkin/Pumpkin1.png"
    },
    "shadow": {
      "cast": true,
      "receive": true
    }
  }
}
```

**Comment by johnchoi313** [comment]
Texture seems to be flipped in the Y axis?

**Comment by nampereira** [comment]
> Texture seems to be flipped in the Y axis?

Fixed in d5f4a92
